service/assessment/assessment.go

```diff
@@ -253,16 +255,14 @@ func (svc *Service) AssessEvidence(_ context.Context, req *assessment.AssessEvid
 		return res, status.Errorf(codes.Internal, "%v", err)
```

Comment:
Shouldn't we return `nil, status.Errorf` here?

Reply:
Also ValidateRequest already returns a status.Error() so we can just `return nil, err` here. otherwise we double-wrap the error.